packages/grid-pro/src/vaadin-grid-pro-edit-column.js

```diff
@@ -129,10 +135,8 @@ class GridProEditColumn extends GridColumn {
   /** @private */
   _cellsChanged() {
     this._cells.forEach((cell) => {
-      const part = cell.getAttribute('part');
-      if (part.indexOf('editable-cell') < 0) {
-        cell.setAttribute('part', `${part} editable-cell`);
-      }
+      const target = cell._focusButton || cell;
+      addValueToAttribute(target, 'part', 'editable-cell');
```

Comment:
This change is needed to make sure the following styles still work:

https://github.com/vaadin/web-components/blob/777750189e5879aa78d7c9c00deeefd1226011f0/packages/grid-pro/theme/lumo/vaadin-grid-pro-styles.js#L25-L27

It's not covered by visual tests, because we only run them on Windows 😕